core/app/models/spree/order.rb

```diff
@@ -583,8 +583,105 @@ def fully_discounted?
     end
     alias_method :fully_discounted, :fully_discounted?
 
+    def add_store_credit_payments
```

Comment:
Assignment Branch Condition size for add_store_credit_payments is too high. [31.21/15]<br>Perceived complexity for add_store_credit_payments is too high. [8/7]